CHANGES.rst

```diff
@@ -26,6 +26,8 @@ Released 2021-05-11
 
 -   Drop support for Python 2.7 and 3.5.
 -   Bump MarkupSafe dependency to >=1.1.
+-   Allow custom autoescape functions :issue:`1377`
```

Comment:
As this won't be a part of 3.0 but 3.1 I will change this and all related docstrings to 3.1